fdbserver/worker.actor.cpp

```diff
@@ -1260,6 +1351,52 @@ ACTOR Future<Void> fdbd(
 	}
 }
 
+ACTOR Future<int> spawnProcess(std::string binPath, vector<std::string> paramList, double maxWaitTime, bool isSync)
```

Comment:
This function should probably be what's actually exported as the API for forking and waiting from platform.cpp

Reply:
Yes, but currently platform has no ACTORs implemented, hence did it this way.

Reply:
Renaming Platform.cpp to Platform.actor.cpp shouldn't be an issue though?

However, I am not sure I agree with Alex here. Should this be in Platform or rather in Net2? Net2 spawns threads after all... I think the nicest solution would be to define this in `INetwork` and implement it in Net2 and sim2.

Reply:
Well, I would prefer to keep platform specific interfaces there. But, I am not fixed on it.

Reply:
My comment here is more that `spawnProcess` seems to present the functionality that anyone would want for launching a subprocess to do a task.  It feels likely to me that anyone that wishes to use `fdbFork` would end up writing the equivalent of `spawnProcess`, so having it be somewhere more discoverable seemed like a good idea.